frontend/micro-ui/web/micro-ui-internals/packages/modules/templates/ApplicationDetails/cashDeskIndex.js

```
@@ -0,0 +1,262 @@
+    }
+    if (nocData !== false && nocMutation) {
+      const nocPrmomises = nocData?.map((noc) => {
+        return nocMutation?.mutateAsync(noc);
+      });
+      try {
+        setIsEnableLoader(true);
+        const values = await Promise.all(nocPrmomises);
+        values &&
+          values.map((ob) => {
+            Digit.SessionStorage.del(ob?.Noc?.[0]?.nocType);
+          });
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)